glean_parser/lint.py

```diff
@@ -555,7 +555,9 @@ def glinter(
     errors += len([nit for nit in nits if nit.check_type == CheckType.error])
 
     if errors == 0:
-        print("✨ Your metrics are Glean! ✨", file=file)
+        if len(objs.value) > 0:
+            # Only report if we parsed some metrics.
+            print("✨ Your metrics are Glean! ✨", file=file)
```

Comment:
Should this fail if no file is provided?

Reply:
I thought about it and put some explanation in the commit message:
No metrics fail is actually allowed behavior. If someone wants to use Glean just for the baseline ping they don't need custom metrics.

Reply:
Do we call glean_parser even though we have no file? I though we didn't even have the integration if the basic integration was all that was desired.

I agree that we should allow empty files.

Reply:
Nevermind, it defaults to `false`

Reply:
Which means I can depend on that. Let me update the PR